### PR TITLE
change HashFor to HashingFor

### PR DESCRIPTION
**Pull request by deepink-mas**
companion for https://github.com/paritytech/substrate/pull/14483

cumulus companion: https://github.com/paritytech/cumulus/pull/2828

### CODE REVIEW

**Comment by paritytech-cicd-pr** [comment]
The CI pipeline was cancelled due to failure one of the required jobs.
**Job name**: test-linux-stable
**Logs**: https://gitlab.parity.io/parity/mirrors/polkadot/-/jobs/3130250

File: node/service/src/lib.rs [hunk 1]
```diff
@@ -100,7 +100,7 @@ pub use sp_api::{ApiRef, ConstructRuntimeApi, Core as CoreApi, ProvideRuntimeApi
 pub use sp_runtime::{
 	generic,
 	traits::{
-		self as runtime_traits, BlakeTwo256, Block as BlockT, HashFor, Header as HeaderT, NumberFor,
+		self as runtime_traits, BlakeTwo256, Block as BlockT, HashingFor, Header as HeaderT, NumberFor,
```
Comment (bkchr) [inline node/service/src/lib.rs:103]:
```suggestion
		self as runtime_traits, BlakeTwo256, Block as BlockT, Header as HeaderT, NumberFor,
```

Not sure we need this here, if this is not used anywhere else.

**Review by bkchr — APPROVED** [review]

**Comment by paritytech-cicd-pr** [comment]
The CI pipeline was cancelled due to failure one of the required jobs.
**Job name**: test-linux-stable
**Logs**: https://gitlab.parity.io/parity/mirrors/polkadot/-/jobs/3136679

**Review by juangirini — APPROVED** [review]

**Comment by ggwpez** [comment]
bot merge

**Pull request merged**